src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center center;
+
+    &:not(:last-child) {
+      margin-right: 0.25rem;
+    }
+  }
+}
+
+.stars--1 .stars__item:nth-child(-n+1),
+.stars--2 .stars__item:nth-child(-n+2),
+.stars--3 .stars__item:nth-child(-n+3),
+.stars--4 .stars__item:nth-child(-n+4),
+.stars--5 .stars__item:nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
try to use cycle for
more detail in [video](https://mate.academy/learn/css-advanced/sass#/video/926)